src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="star"></div>
```

Comment:
Fix classes everywhere
```suggestion
      <div class="stars__star"></div>
```